torchrl/envs/libs/vmas.py

```diff
@@ -115,13 +115,20 @@ class VmasWrapper(_EnvWrapper):
         env (``vmas.simulator.environment.environment.Environment``): the vmas environment to wrap.
 
     Keyword Args:
-        num_envs (int): Number of vectorized simulation environments.
-        device (torch.device, optional): Device for simulation.
-        continuous_actions (bool, optional): Weather to use continuous actions. Defaults to ``True``.
-        max_steps (int, optional): Maximum number of steps in each vectorized environment after which done is returned.
-            Defaults to ``None`` (no truncation).
+        num_envs (int): Number of vectorized simulation environments. VMAS perfroms vectorized simulation using PyTorch.
+            This argument decides the number of vectorized envuironments that should be simulated in a batch. It will also
+            determine the batch size of the environment.
+        device (torch.device, optional): Device for simulation. Defaults to ``"cpu"``. All the tensors created by VMAS
+            will be placed on this device.
+        continuous_actions (bool, optional): Whether to use continuous actions. Defaults to ``True``. If ``False``, actions
+            will be discrete. The number of actions and their size will depend on the scenario chosen.
+            See the VMAS repositiory for more info.
+        max_steps (int, optional): Horizon of the task. Defaults to ``None`` (infinite horizon). Each VMAS scenario can
+            implement a ``done`` function that will define when the scenario is terminated. If ``max_steps`` is specified,
```

Comment:
what does it mean that an env can define a done funtion? Some envs are never done? Can I choose if they have a done function?

Reply:
Yes, each scenario has a optional "done" function. Some scenarios do not present that yes. Those scenarios would be eventually terminated if you use max_steps. You can choose if you implement a scenario yourself, otherwise it is a property of the scenario

Reply:
got it
What is the `done` function in practice?
How does it relate to the `"done"` key?
I'm asking this to clarify if it's required to mention a function that we do not see appearing anywhere or if we can just say that env can be non-terminating unless max_steps is provided.

Reply:
I am also pro to not mention it. I just did cause I did not know how much depth is required here.

we can definitely just say what you proposed. Seems better to me to.

We can say something like: some scnearios are terminating and some not. In all cases max_steps provides an additional termination condition

Reply:
I don't think users need to know the inner machinery but they should know precisely how changing a kwarg will impact what they get from the environment.

Reply:
Changed